### PR TITLE
5.4.0-tiup-deployment-change

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: production-deployment-using-tiup.md [hunk 1]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+Started cluster `tidb-test` successfully.
+The root password of TiDB database has been changed.
+The new password is: 'y_+3Hwp=*AWz8971s6'.
+Copy and record it to somewhere safe, it is only displayed once, and will not be stored.
+The generated password can NOT be get and shown again.
+
```
Comment (qqqdan) [inline production-deployment-using-tiup.md:372]:
```

File: production-deployment-using-tiup.md [hunk 2]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
 
 ## 第 7 步：启动集群
 
+> **注意：**
+>
+> 安全启动方式是对 v5.4.0 以上版本新提供的部署方式，推荐使用安全启动方式启动集群：
```
Comment (AstroProfundis) [inline production-deployment-using-tiup.md:346]: 目前的实现是兼容所有 TiDB 版本的，并不局限于 5.4 以上

Comment (qqqdan) [reply production-deployment-using-tiup.md]: 已修改

**Review by AstroProfundis — APPROVED** [review]

File: production-deployment-using-tiup.md [hunk 3]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
...
 
+> **注意：**
+>
+> 安全启动方式是在 tiup cluster v1.9.0 及后续版本中提供的新部署方式，推荐使用安全启动方式启动集群：
+>
+> - 安全启动后会自动生成 root 用户的密码，并在命令行界面返回密码。
+>
+> - 使用安全启动方式后，不能通过无密码的 root 用户登录数据库，用户需要记录命令行返回的密码进行后续操作。
+>
+> - 该自动生成的密码只会返回一次，如果用户没有记录或者忘记该密码，请参照 [忘记 root 密码](https://docs.pingcap.com/zh/tidb/stable/user-account-management#%E5%BF%98%E8%AE%B0-root-%E5%AF%86%E7%A0%81) 修改密码。
```
Comment (shichun-0415) [inline production-deployment-using-tiup.md:352]:
```suggestion
> - 该自动生成的密码只会返回一次，如果用户没有记录或者忘记该密码，请参照[忘记 root 密码](https://docs.pingcap.com/zh/tidb/stable/user-account-management#%E5%BF%98%E8%AE%B0-root-%E5%AF%86%E7%A0%81) 修改密码。
```

File: production-deployment-using-tiup.md [hunk 4]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
...
+预期结果如下，表示启动成功。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+Started cluster `tidb-test` successfully.
+The root password of TiDB database has been changed.
+The new password is: 'y_+3Hwp=*AWz8971s6'.
+Copy and record it to somewhere safe, it is only displayed once, and will not be stored.
+The generated password can NOT be get and shown again.
```
Comment (shichun-0415) [inline production-deployment-using-tiup.md:371]:
```suggestion
The generated password can NOT be get and shown again.
```
A grammar error in “can NOT be get”. It should be "cannot be got/obtained".

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: production-deployment-using-tiup.md (hunk 3 above)
Comment (TomShawn) [reply production-deployment-using-tiup.md:352]:
```suggestion
> - 该自动生成的密码只会返回一次，如果用户没有记录或者忘记该密码，请参照[忘记 root 密码](/user-account-management.md#忘记-root-密码) 修改密码。
```

File: production-deployment-using-tiup.md [hunk 5]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
 
 ## 第 7 步：启动集群
 
+> **注意：**
+>
+> 安全启动方式是在 tiup cluster v1.9.0 及后续版本中提供的新部署方式，推荐使用安全启动方式启动集群：
```
Comment (qiancai) [inline production-deployment-using-tiup.md:346]:
```suggestion
> 安全启动方式是为 TiUP cluster v1.9.0 及后续版本中提供的新部署方式，推荐使用安全启动方式启动集群：
```

File: production-deployment-using-tiup.md [hunk 6]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
...
+>
+> - 安全启动后会自动生成 root 用户的密码，并在命令行界面返回密码。
```
Comment (qiancai) [inline production-deployment-using-tiup.md:348]:
```suggestion
> - 安全启动后会自动生成 TiDB root 用户的密码，并在命令行界面返回密码。
```

File: production-deployment-using-tiup.md (hunk 4 above)
Comment (qqqdan) [reply production-deployment-using-tiup.md]: 已修改

File: production-deployment-using-tiup.md [hunk 7]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
...
+>
+> - 使用安全启动方式后，不能通过无密码的 root 用户登录数据库，用户需要记录命令行返回的密码进行后续操作。
```
Comment (qiancai) [inline production-deployment-using-tiup.md:350]:
```suggestion
> - 使用安全启动方式后，不能通过无密码的 root 用户登录数据库，你需要记录命令行返回的密码进行后续操作。
```

File: production-deployment-using-tiup.md [hunk 8]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
...
+>
+> - 该自动生成的密码只会返回一次，如果用户没有记录或者忘记该密码，请参照[忘记 root 密码](/user-account-management.md#忘记-root-密码)修改密码。
```
Comment (qiancai) [inline production-deployment-using-tiup.md:352]:
```suggestion
> - 该自动生成的密码只会返回一次，如果没有记录或者忘记该密码，请参照[忘记 root 密码](/user-account-management.md#忘记-root-密码)修改密码。
```

File: production-deployment-using-tiup.md [hunk 9]
```diff
@@ -341,13 +341,47 @@ tiup cluster display tidb-test
...
+
+```shell
+Started cluster `tidb-test` successfully.
+The root password of TiDB database has been changed.
+The new password is: 'y_+3Hwp=*AWz8971s6'.
+Copy and record it to somewhere safe, it is only displayed once, and will not be stored.
+The generated password can NOT be get and shown again.
+```
+
+用户需要记录命令行返回结果中的密码信息，用于后续数据库登录。使用安全启动方式后，不能通过无密码的 root 用户登录数据库。
```
Comment (qiancai) [inline production-deployment-using-tiup.md:374]:
```suggestion
你需要记录命令行返回结果中的密码信息，用于后续数据库登录。使用安全启动方式后，不能通过无密码的 root 用户登录数据库。
```

File: production-deployment-using-tiup.md [hunk 10]
```diff
@@ -341,13 +341,45 @@ tiup cluster display tidb-test
 
 ## 第 7 步：启动集群
 
+安全启动方式是为 TiUP cluster v1.9.0 及后续版本提供的新部署方式，采用该模式登录数据库可以提高数据库安全性。推荐使用安全启动方式启动集群。
```
Comment (qqqdan) [inline production-deployment-using-tiup.md:344]: 安全启动方式是 TiUP cluster v1.9.0 开始提供的新部署方式，采用该模式启动数据库可以提高数据库安全性。

## 说明，这个方式是 TiUP cluster 从 v1.9.0 版本开始提供的。

File: production-deployment-using-tiup.md [hunk 11]
```diff
@@ -341,13 +341,45 @@ tiup cluster display tidb-test
...
+方式二：普通启动方式
+
 {{< copyable "shell-regular" >}}
 
 ```shell
 tiup cluster start tidb-test
 ```
 
-预期结果输出 ```Started cluster `tidb-test` successfully``` ，表示启动成功。
+预期结果输出 ```Started cluster `tidb-test` successfully``` ，表示启动成功。使用普通启动方式后，可通过无密码的 root 用户登录数据库。
```
Comment (qiancai) [inline production-deployment-using-tiup.md:382]:
```suggestion
预期结果输出 ```Started cluster `tidb-test` successfully```，表示启动成功。使用普通启动方式后，可通过无密码的 root 用户登录数据库。
```

File: production-deployment-using-tiup.md (hunk 10 above)
Comment (qiancai) [inline production-deployment-using-tiup.md:344]:
```suggestion
安全启动方式是为 TiUP cluster v1.9.0 及后续版本提供的新部署方式，采用该方式登录数据库可以提高数据库安全性。推荐使用安全启动方式启动集群。
```

File: production-deployment-using-tiup.md [hunk 12]
```diff
@@ -341,13 +341,45 @@ tiup cluster display tidb-test
 
 ## 第 7 步：启动集群
 
+安全启动是 TiUP cluster v1.9.0 引入的一种新的启动方式，采用该方式启动数据库可以提高数据库安全性。推荐使用安全启动。
```
Comment (qiancai) [inline production-deployment-using-tiup.md:344]:
```suggestion
安全启动是 TiUP cluster 从 v1.9.0 起引入的一种新的启动方式，采用该方式启动数据库可以提高数据库安全性。推荐使用安全启动。
```

File: production-deployment-using-tiup.md [hunk 13]
```diff
@@ -341,13 +341,45 @@ tiup cluster display tidb-test
...
+
+安全启动后，TiUP 会自动生成 TiDB root 用户的密码，并在命令行界面返回密码。
+
+> **注意：**
+>
+> - 使用安全启动方式后，不能通过无密码的 root 用户登录数据库，你需要记录命令行返回的密码进行后续操作。
+>
+> - 该自动生成的密码只会返回一次，如果没有记录或者忘记该密码，请参照[忘记 root 密码](/user-account-management.md#忘记-root-密码)修改密码。
+
+方式一：安全启动方式
```
Comment (qiancai) [inline production-deployment-using-tiup.md:354]:
```suggestion
方式一：安全启动
```

File: production-deployment-using-tiup.md [hunk 14]
```diff
@@ -341,13 +341,45 @@ tiup cluster display tidb-test
...
+
+```shell
+Started cluster `tidb-test` successfully.
+The root password of TiDB database has been changed.
+The new password is: 'y_+3Hwp=*AWz8971s6'.
+Copy and record it to somewhere safe, it is only displayed once, and will not be stored.
+The generated password can NOT be got again in future.
+```
+
+方式二：普通启动方式
```
Comment (qiancai) [inline production-deployment-using-tiup.md:374]:
```suggestion
方式二：普通启动
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**